### PR TITLE
Update threat-explorer.md

### PR DESCRIPTION
**Pull request by andypunt**
Updated real-time detections with Real-time detections

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/threat-explorer.md [hunk 1]
```diff
@@ -172,9 +172,9 @@ Also, you will now be able to export reports for URL clicks to Microsoft Excel i
...
 When you click on a URL in the list, you'll see a new Export button on the fly-out panel. Use this button to move data to an Excel spreadsheet for easier reporting.
 
-You can get to the same location in the real-time detections report as follows:
+You can get to the same location in the Real-time detections report as follows:
 
-**Explorer** \> **Real-time Detections** \> **View Phish** \> **URLs** \> **Top URLs or Top Clicks** \> **Click on any record to open URL flyout** \> **Navigate to the Clicks Tab.**
+**Explorer** \> **Real-time detections** \> **View Phish** \> **URLs** \> **Top URLs or Top Clicks** \> **Click on any record to open URL flyout** \> **Navigate to the Clicks Tab.**
 
 > [!TIP]
 > Network Message ID maps the click back to specific mails when you search through Explorer or associated 3rd party tools via Network Message ID. Searching through the Network Message ID will give admins the specific email associated with a click result. On export having, the correlating identification of Network Message ID makes for quicker and more powerful analysis.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
> Network Message ID maps the click back to specific mails when you search through Explorer or associated third-party tools via Network Message ID. Searching through the Network Message ID will give admins the specific email associated with a click result. On export, having the correlating identification of Network Message ID makes for quicker and more powerful analysis.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - two additional minor edits, please.

**Comment by yogkumgit** [comment]
@andypunt Please implement the review comments. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by MSFTTracyP — APPROVED** [review]
Update

**Pull request merged**